src/f3dasm/_src/experimentdata/_data.py

```diff
@@ -291,6 +291,23 @@ def n_best_samples(self, nosamples: int, column_name: List[str] | str) -> pd.Dat
         """
         return self.data.nsmallest(n=nosamples, columns=column_name)
 
+    def select_columns(self, columns: Iterable[str] | str) -> _Data:
+        """Filter the data on the selected columns.
+
+        Parameters
+        ----------
+        columns : Iterable[str] | str
+            The columns to select.
+
+        Returns
+        -------
+        _Data
+            The data only with the selected columns
+        """
+        # This is necessary otherwise self.data[columns] will be a Series
+        if isinstance(columns, str):
+            columns = [columns]
+        return _Data(self.data[columns])
```

Comment:
A method that returns a private object should be private as well.

Reply:
Since the whole `_Data` object is a private object, I decided to not prefix all methods with an underscore since it is in my view redundant

Reply:
Makes sense!